server/schema/schema.json

```diff
@@ -0,0 +1,69 @@
+"users": 

```

Comment:
I asked in the wrong PR, but what is the benefits of having both each schema and a container schema?

Reply:
what do you mean @JeyzerMC

Reply:
Like what is this specific file used for? It's a concatenation of all the other schemas, that's why

Reply:
It was easier to share with the rest of the team. My shell script requires them to be different files to mock the data though so I added both.